app/src/main/java/com/example/ikeahjalpare/dao/IFurnitureDAO.kt

```diff
@@ -4,11 +4,13 @@ import com.example.ikeahjalpare.dto.Furniture
 import retrofit2.Call
 import retrofit2.http.GET
 
+/*
+ * Retrieves a list of furniture from IKEA using the specified endpoints
+ */
```

Comment:
This is an example of the kDoc comments that I mention in the main PR

Reply:
Good.